kafka-ui-react-app/src/components/ConsumerGroups/List/List.tsx

```diff
@@ -27,36 +71,31 @@ const List: React.FC = () => {
           handleSearch={handleInputChange}
         />
       </ControlPanelWrapper>
-      <Table isFullwidth>
-        <thead>
-          <tr>
-            <TableHeaderCell title="Consumer Group ID" />
-            <TableHeaderCell title="Num Of Members" />
-            <TableHeaderCell title="Num Of Topics" />
-            <TableHeaderCell title="Messages Behind" />
-            <TableHeaderCell title="Coordinator" />
-            <TableHeaderCell title="State" />
-          </tr>
-        </thead>
-        <tbody>
-          {consumerGroups
-            .filter(
-              (consumerGroup) =>
-                !searchText || consumerGroup?.groupId?.indexOf(searchText) >= 0
-            )
-            .map((consumerGroup) => (
-              <ListItem
-                key={consumerGroup.groupId}
-                consumerGroup={consumerGroup}
-              />
-            ))}
-          {consumerGroups.length === 0 && (
-            <tr>
-              <td colSpan={10}>No active consumer groups</td>
-            </tr>
-          )}
-        </tbody>
-      </Table>
+      <SmartTable
+        tableState={tableState}
+        isFullwidth
+        placeholder="No active consumer groups"
+        hoverable
+      >
+        <TableColumn
+          title="Consumer Group ID"
+          cell={GroupIDCell}
+          orderValue={ConsumerGroupOrdering.NAME}
+        />
+        <TableColumn
+          title="Num Of Members"
+          field="members"
+          orderValue={ConsumerGroupOrdering.MEMBERS}
+        />
+        <TableColumn title="Num Of Topics" field="topics" />
+        <TableColumn title="Messages Behind" field="messagesBehind" />
+        <TableColumn title="Coordinator" cell={CoordinatorCell} />
```

Comment:
please pass field prop with value "coordinator.id" and remove custom component for this column